src/main/java/org/folio/mr/domain/entity/SecureRequestEntity.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.mr.domain.entity;
+
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.folio.mr.domain.converter.UUIDConverter;
+import java.util.UUID;
+
+@Entity
+@Table(name = "secure-requests")
```

Comment:
Observing other modules, it appears that we typically use singular table names with underscore word separation, for example: "secure_request." It might be appropriate to follow this convention here

Reply:
Sure, seems typo